regolith/validators.py

```diff
@@ -109,4 +116,35 @@ def ensure_email(email):
     'databases': (always_false, ensure_databases),
     'stores': (always_false, ensure_stores),
     'email': (always_false, ensure_email),
-    }
+}
+
+
+def pop_description(schema):
+    """Pop description off the schema since Cerberus doesn't support it"""
+    if isinstance(schema, dict):
+        schema.pop('description', None)
+        for v in schema.values():
+            pop_description(v)
```

Comment:
You shouldn't need to do this. Instead subclass validtor and make it skip custom keys in the schema we want to add.

Reply:
See http://docs.python-cerberus.org/en/stable/customize.html#custom-validators

Reply:
I spent some time trying to do that without much headway, but I will try again.

Reply:
There has to be a way.  Popping stuff out of a global, mutable object is wrong.

Reply:
In this case at least. The alternative is to have a second dict which has the same structure and only contains the descriptions, but that feels wrong too.

Reply:
Btw there is an open issue for `description` https://github.com/pyeve/cerberus/issues/254

Reply:
And now a SO question https://stackoverflow.com/questions/47851546/cerberus-additional-schema-keys